kornia/augmentation/container/ops.py

```diff
@@ -195,6 +197,18 @@ def inverse(cls, input: Tensor, module: Module, param: ParamItem, extra_args: Di
         return input
 
 
+class NoOps(SequentialOpsInterface[Tensor]):
```

Comment:
Can you change it to maybe `ClassSequentialOps` or something similar? you may detect the module and throw a RuntimeError when `isinstance(module, (MixAugmentationV2, MixAgumentation,))`.

BTW, we do have processors for the labels for mix augmentations, like in https://github.com/kornia/kornia/blob/4cd9a61f50c4661d829a15703ca86260012d30e2/kornia/augmentation/_2d/mix/mixup.py#L98-L124. 

But probably throw an error for now for simplicity.

Reply:
I actually looked a bit into it after submitting the PR and it made me a bit uneasy because you are passing in labels in one format and they come out in a different one. Don't many things break in longer pipelines where there might be more augmentations that do not consider this?

Reply:
For mixup ops, we need also to check if it is one-hot classes or not. The operation returns three vectors as I remember, and the first one shall be the classes to return if I remember correctly.

While I made it returns three vectors since the Mixup op needs a new loss computation that needs the lambda values. If only to consider making the `AugmentationSequential` work properly, returning the first one shall be enough. The problem is to find the best practices to use the lambda values afterward.